src/main/docs/guide/httpServer/customBinding.adoc

```diff
@@ -0,0 +1,43 @@
+
+== Argument binding
+
+An argument binding allows for the binding of an argument's value from a source.
+Micronaut follows a specific search path for the resolution of api:core.bind.ArgumentBinder[ArgumentBinder]s through the
+api:http.bind.binders.ArgumentBinderRegistry[ArgumentBinderRegistry].  The selection process starts with annotation binders
+and then searches "by type" binders.  Selection of a binder can also be affected by other factors such as the request media
+type.
+
+Unlike api:core.convert.TypeConverter[TypeConverter] instances, api:core.bind.ArgumentBinder[ArgumentBinder]s can potentially
+handle complex objects and typically work in conjunction with a api:core.convert.value.ConvertibleValues[ConvertibleValues] instance
+
+An api:core.bind.ArgumentBinder[ArgumentBinder] can either be registered as a bean or by META-INF/services. In the case
```

Comment:
Is that true? Can you point me to where in the code that is happening?

Reply:
@jameskleeh I got this from the documentation of the interface ArgumentBinder